api/v1alpha1/releaselink_types.go

```diff
@@ -34,7 +47,7 @@ type ReleaseLinkSpec struct {
 	// Target is a reference to the workspace to establish a link with
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
 	// +required
-	Target string `json:"target"`
+	Target Target `json:"target"`
```

Comment:
This should be renamed to `TargetReleaseLink` so it's more clear. This will be done as part of a future PR to split the ReleaseLink in two.

Reply:
I believe we said offline not to rename here. Feel free to unresolve if that is not the case

Reply:
Yeah, that's what my comment here says :stuck_out_tongue_closed_eyes: